### DISCUSSION TITLE
774

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#945] [Comments]
# 774

코인 노래방을 종종 다닌다. 스트레스를 풀기위해 다니는데 노래를 못해서 낮은 노래를 선택하고 음이탈이 나지 않으려고 최대한 조심해서 불러왔다. 스트레스를 풀러 간 어느 날, 노래방 밖에서도 들릴 정도로 엄청 큰 소리로 노래를 부르는 사람이 있었다. 음정 박자도 전혀 맞지 않고, 음이탈이 나도 그냥 기세로 밀어붙이면서 노래를 불렀다. 밖에 있는데도 듣기 힘들 정도로 고함치는 것에 가깝게 노래를 부르던 그 사람은 노래를 마치고 아주 개운한 표정으로 방에서 빠져나왔다. 남들이 어떻게 생각하든 본인은 아주 만족한 채로 노래를 마친 표정이였다. 그 분의 표정을 보면서 그간 부르고 싶은 노래 보다는 음역대가 맞는 노래에 맞춰 음이탈이 안나는 것에 급급하여 살살 부르던 것이 생각나 갑자기 짜증이 확 났다. "스트레스..

https://jojoldu.tistory.com/774

**Comment by bangjaeyoung** [comment]
일상적인 이야기로 공감을 이끌어내주셔서 감사합니다 :)
